### PR TITLE
Add getting started documentation

### PR DESCRIPTION
**Pull request by mravn-google**

### CODE REVIEW

File: packages/google_maps_flutter/README.md [hunk 1]
```diff
@@ -44,4 +44,75 @@ as a [dependency in your pubspec.yaml file](https://flutter.io/platform-plugins/
...
+              ),
+            )
+          );
+        }
+      ),
+    ]);
+  }
+```
+
 See the `example` directory for a complete sample app using Google Maps.
```
Comment (sigurdm) [inline packages/google_maps_flutter/README.md]: Consider deleting "using Google Maps" in this last line

File: packages/google_maps_flutter/README.md [hunk 2]
```diff
@@ -44,4 +44,75 @@ as a [dependency in your pubspec.yaml file](https://flutter.io/platform-plugins/
 
 ## Getting Started
 
-See the `example` directory for a complete sample app using Google Maps.
+Get an API key at <https://cloud.google.com/maps-platform/>.
+
+### Android
+
+Specify your API key in the application manifest `android/src/main/AndroidManifest.xml`:
```
Comment (mit-mit) [inline packages/google_maps_flutter/README.md]: /android/app/src/... ?

File: packages/google_maps_flutter/README.md [hunk 3]
```diff
@@ -44,4 +44,75 @@ as a [dependency in your pubspec.yaml file](https://flutter.io/platform-plugins/
...
+You can now instantiate a `GoogleMapOverlayController` and use it to configure
+a `GoogleMapOverlay` widget. Client code will have to change once the plugin
+stops using platform overlays.
+
+Once added as an overlay, the map view can be controlled via the
+`GoogleMapController` that you obtain as the `mapController` property of
+the overlay controller. Client code written against the `GoogleMapController`
+interface will be unaffected by the change away from platform overlays.
+
+```dart
```
Comment (mit-mit) [inline packages/google_maps_flutter/README.md]: Also show the import?

```
import 'package:google_maps_flutter/google_maps_flutter.dart';
```

**Comment by mravn-google** [comment]
@mit-mit Thanks for the review. Comments addressed in https://github.com/flutter/plugins/pull/554.

**Pull request merged**